great_expectations/expectations/expectation.py

```diff
@@ -329,7 +331,11 @@ class Expectation(metaclass=MetaExpectation):
         2. Data Docs rendering methods decorated with the @renderer decorator. See the
     """
 
-    version: ClassVar = ge_version
+    class Config:
+        arbitrary_types_allowed = True
+        extra = "allow"
```

Comment:
Doesn't feel great but a necessary setting to keep until we start adding defined params through the inheritance hierarchy